.github/scripts/release/build-linux-release.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# This is used to build our binaries:
+# - polkadot
+# - polkadot-parachain
+# set -e
+
+BIN=$1
+PACKAGE=${2:-$BIN}
+
+PROFILE=${PROFILE:-production}
+RUST_TOOLCHAIN=stable
```

Comment:
Can we get this from the global CI env file? Otherwise older releases will stop building when newer rust versions come out.

Generally it is good to have all the versions fixed in files on the release branch, so that they continue to work in the future even when dependencies update.

Reply:
It should always be the same version that we use in the CI to run tests, as only that version is guaranteed to work.